docs/source/concept_models/vivarium_swissre_cervicalcancer/cervical_cancer_concept_model.rst

```diff
@@ -0,0 +1,628 @@
+
+:underline:`II. Screening schedule and attendance`
+
+Probability of attending screening
+ - Generate 1000 draws from normal distribution with mean=0.2, SD=0.002 for
+   calculating the probability of simulants attending their first due screening.
+ - If a simulant attended their last screening, they have 1.89 with 95%CI 1.06-2.49
+   (Yan et al. 2017) more odds of attending the next screening than those who did
+   not attend their last screening. 
+Time to next scheduled screening
+ - For those who are in Branch A (yearly cytology): use truncated normal
+   distribution with mean 364 days, SD +/- 148 days, lower limit is 100 days,
+   upper limit is 700 days.
+ - For those in Branch B (every 3 years cytology): use truncated normal
+   distribution with mean 1092 days, SD +/- 148 days, lower limit is 300 days,
+   upper limit is 2100 days.
+
+:underline:`III. Screening initialization`
+
+The date of the first screening appointment (T_appt) for females at age between 
+21 and 65 is determined as follows. We assume that each simulant had a previous 
+appointment scheduled at some point before the simulation begins. We calculate 
+the time between that past appointment and their next appointment (delta_T) using 
+the methodology outlined in Section 5.3.2.II (Time to next scheduled screening). 
+With a uniform distribution we randomly determine how far along that time interval 
+between appointments each individual is (X) at the beginning of the simulation (
+T_0). For females under 21 when the simulation begins the methodology is identical, 
+except T_0 is the simulant's 30th birthday rather than the beginning of the 
+simulation. No screening appointment will be initialized for males at any age, as 
+well as females above 65.
+
+.. image:: cervical_cancer_screening_event_time.svg
+
+.. _5.3.3:
+
+5.3.3 Risk factor model
+~~~~~~~~~~~~~~~~~~~~~~~
+GBD risk factors: N/A
+
+Non-GBD risk factor: Human Papilloma Virus (HPV)
+ - prevalence: 19.0% with 95%CI 17.1-20.9 (Li et al. 2019)
+ - Incidence: TBD
+ - exposure distirbution: dichotomous
+ - relative risk: RR = 16.2 with 95%CI 9.6 to 27.3 (Chen et al. 2011)
+
+relevant formulas 
+ (1) PAF = :math:`\frac{\text{Prev_HPV}(RR-1)}{\text{Prev_HPV}(RR-1)+1}`
+ (2) :math:`\text{i_HPV+} =  i \times (1-PAF) \times RR`
+ (3) :math:`\text{i_HPV-} =  i \times (1-PAF)`
+
+.. note::
+
+  - Find HPV incidence (age-specific or age-standardized) from external sources
+  - How can we include the change of HPV incience by year?
+
+.. _5.3.4:
+
+5.3.4 Treatment model
+~~~~~~~~~~~~~~~~~~~~~
+Treatment for benign cervical cancer
+
+.. todo::
+
+ add more details
+
+.. _5.4:
+
+5.4 Input data sources
+----------------------
+
+.. list-table:: Model inputs
+   :header-rows: 1
+
+   * - Input parameter
+     - Value
+     - Source
+     - Note
+   * - Duration from BCC to CC
+     - 14.5 years
```

Comment:
I think documenting the uncertainty distributions around these parameters (or indicating if you plan to use point estimates) would be helpful so that it's good to go when the software engineers need to implement.

Reply:
We aim to add uncertainty bound once found more China-specific studies. Alternatively we could do a triangular distribution with mean of 10 years, min and max to be 5 years and 15 years according to data identified from different countries.